nixos/modules/services/web-apps/zabbix.nix

```diff
@@ -215,10 +254,28 @@ in
             Options -Indexes
             DirectoryIndex index.php
           </Directory>
-        '';
+          '';
       } ];
     };
 
+    services.nginx = mkIf (cfg.frontend == "nginx") {
+        enable = true;
+        virtualHosts.${cfg.hostname} = mkMerge [
```

Comment:
i'm guessing `zabbix` documentation includes specific instructions on what to do for a `nginx` reverse proxy... is that where you got these instructions?

Reply:
No, it doesn't (or I couldn't find any official documentation about it). It basically ports the existing apache config to an nginx config. 
There are a bunch of inofficial forum posts about it, but this (as well as the preexisting apache config) is a minimal working example to display the `php` server files

Reply:
thanks for doing the leg work then!